synapse/replication/http/_base.py

```diff
@@ -196,6 +196,9 @@ async def send_request(instance_name="master", **kwargs):
                     except CodeMessageException as e:
                         if e.code != 504 or not cls.RETRY_ON_TIMEOUT:
                             raise
+                    except RequestTimedOutError:
```

Comment:
RequestTimedOutError is a CodeMessageException, so this won't get hit (and is redundant)

Reply:
Gah, I'm getting confused with how the federation client works :/

Reply:
Thanks